storefront/app/controllers/workarea/storefront/current_checkout.rb

```diff
@@ -26,7 +26,10 @@ def current_order
       #
       def current_order=(order)
         @current_order = order
-        cookies.permanent.signed[:order_id] = order&.id
+
+        if order.blank? || order.persisted?
```

Comment:
I am confused by this. Did you mean !order.blank?

Reply:
Or maybe it is a way to unset it when you need to?

Reply:
Yeah, it was intentional because if you set `current_order = nil`, we want to clear out that cookie. There is some behavior that relies on that.